src/lib/openapi/spec/client-metrics-env-schema.ts

```diff
@@ -9,28 +9,46 @@ export const clientMetricsEnvSchema = {
     properties: {
         featureName: {
             type: 'string',
+            description: 'Name of the feature checked by the SDK',
+            example: 'my.special.feature',
         },
         appName: {
+            description: 'The name of the application the SDK is being used in',
             type: 'string',
+            example: 'accounting',
         },
         environment: {
+            description: 'Which environment the SDK is being used in',
             type: 'string',
+            example: 'development',
         },
         timestamp: {
+            description: 'The time window these metrics are valid for',
```

Comment:
Time window makes me think about a range of dates, but the timestamp is just a date. Is this the start date of the window and is the end date: `start date + push metrics interval`? I think it's worth clarifying this

Reply:
Good point. Yes. Will clarify